tmux.conf

```diff
@@ -33,7 +31,8 @@ set -g status-right ''
 set -g history-limit 10000
 
 # switch to last pane
-bind-key C-a last-pane
+bind-key C-b last-pane
+bind-key C-s last-pane
```

Comment:
@christoomey do you know if there's a `prefix` type of command so we don't need to specify both `C-b` and`C-s` here?